File: customer-service/src/main/java/com/athena/customerservice/dto/MediaResponse.java
package com.athena.customerservice.dto;

import lombok.Data;
import java.util.UUID;

@Data
public class MediaResponse {
    private UUID id;
    private String mediaType;
    private String originalFilename;
    private String contentType;
    private Long fileSize;
    private String serviceName;
    private String channel;
}

File: customer-service/internal/dto/media_response.go
package dto

// MediaResponse mirrors the Java MediaResponse DTO returned by media-service.
type MediaResponse struct {
	ID               string `json:"id"`
	MediaType        string `json:"mediaType"`
	OriginalFilename string `json:"originalFilename"`
	ContentType      string `json:"contentType"`
	FileSize         int64  `json:"fileSize"`
	ServiceName      string `json:"serviceName"`
	Channel          string `json:"channel"`
}
